src/main/resources/test_files/runtime/FunctionSqrt.iq

```diff
@@ -0,0 +1,6 @@
+(:JIQS: ShouldRun; Output="( 0.0, -0.0, 1000.0)" :)
+sqrt(0.0e0),
```

Comment:
It would be great to add a test for sqrt of a negative number (double-checking which error should be returned in the specification)